### PR TITLE
Add Levels

### LINKED ISSUE #4: Define levels
For the game to be interesting we must let the user notice their progress, for this we will make their level within the game increase.

It should be considered:

- [x] Change the level with a certain amount of lines created
- [x] Increase speed when level change
- [x] Show current level

### PR DESCRIPTION
**Pull request by iguanajazz**
For the game to be interesting we must let the user notice their progress, for this we will make their level within the game increase.

It should be considered:

- [x] Change the level with a certain amount of lines created
- [x] Increase speed when level change
- [x] Show current level

_____________________________________________________________________________________________________________________
Increase the level of game when a specified amount of lines is reached and speed up every level but when level 15 is reached (configurable) the speed is slow down to the reset speed as a reward

Closes #4

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -5,9 +5,11 @@
 GRID_HEIGHT = NUM_COLUMNS * TILE_SIZE
 WINDOW_WIDTH = GRID_WIDTH + PANEL_SIZE
 WINDOW_HEIGHT = GRID_HEIGHT + 1
-INITIAL_FALL_SPEED = 1000
+INITIAL_FALL_SPEED = 900
```
Comment (clausia) [inline src/constants.py]: This should be 1000 (1 second)

File: src/tetromiq.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def game():
     paused = False
     game_over = False
     fall_speed = INITIAL_FALL_SPEED
-    level = 1
+    previous_level = 1
```
Comment (clausia) [inline src/tetromiq.py:17]: Why not only `level = 1`?

File: src/tetromiq.py [hunk 3]
```diff
@@ -97,23 +99,46 @@ def game():
...
-        if blocks.score >= SCORE_CHANGE_LEVEL * level:
-            level += 1
+        
+       
+    pygame.quit()
+
+
+def update_fall_speed(blocks, fall_speed, previous_level, EVENT_UPDATE_CURRENT_BLOCK):
+    # change falling speed based on level
+    #if blocks.level >= LINES_CHANGE_SPEED_LEVEL * speed_level:
```
Comment (clausia) [inline src/tetromiq.py]: Don't leave unused code commented, please delete

**Review by clausia — CHANGES_REQUESTED** [review]
Only a few comments on the code

File: src/tetromiq.py (hunk 2 above)
Comment (iguanajazz) [reply src/tetromiq.py:17]: I use `previous_level` to determine if I need to update the falling speed, so is used to know if the level changed

File: src/constants.py (hunk 1 above)
Comment (iguanajazz) [reply src/constants.py]: done

File: src/tetromiq.py (hunk 3 above)
Comment (iguanajazz) [reply src/tetromiq.py]: yes, sorry

**Review by clausia — APPROVED** [review]
looks good to me

**Review by Isamade — APPROVED** [review]
Works perfectly!

**Review by Isamade — APPROVED** [review]

**Pull request merged**